api/python/provisioner/commands/validate_setup.py

```diff
@@ -134,12 +134,27 @@ def __attrs_post_init__(self):
         if len(missing_params) > 0:
             raise ValueError(f"Mandatory param missing {missing_params}")
 
+@attr.s(auto_attribs=True)
+class StorageNodeParamsValidation:
+    hostname: str = StorageNodeParams.hostname
```

Comment:
This is only for testing and a placeholder for future node-specific storage enclosure params.